### PR TITLE
fix HTML titles are now displayed in HTML tag format.

### LINKED ISSUE #644: dog page Searcher Modal title bug
_ErrorBoundary__Component_-_Blitz_js_-_Vivaldi-2
Probably "&gt;" is displayed where "<" or ">" should be displayed.

### PR DESCRIPTION
**Pull request by mochi-sann**
fix #644 
## before 
image
## after
Future_of_Blitz_-_Vivaldi
r

### CODE REVIEW

File: app/core/components/Search.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ export function Search({className = ""}) {
     },
     [setIsOpen, setInitialQuery],
   )
+  const ConvertToHtmlTag = (value) => {
+    return value.replace(/&amp;/g, "&").replace(/&lt;/g, "<").replace(/&gt;/g, ">")
+  }
```
Comment (beerose) [inline app/core/components/Search.js:42]: Let's keep the function name lowercase
```suggestion

  const convertToHtmlTag = (value) => {
    return value.replace(/&amp;/g, "&").replace(/&lt;/g, "<").replace(/&gt;/g, ">")
  }
```

**Review by beerose — COMMENTED** [review]
Thanks!

**Comment by mochi-sann** [comment]
Thank you for your review!

**Review by JuanM04 — CHANGES_REQUESTED** [review]
Nice!
I noticed that `hierarchy.lvl0` doesn't need to be converted anymore, so the code can be reduced to this:

```js
// The titles are encoded, so we need to decode them.
// &lt;Script&gt; --> <Script>
const _highlightResult = {...item._highlightResult}

_highlightResult.hierarchy.lvl0.value = _highlightResult.hierarchy.lvl0.value
  .replace(/&amp;/g, "&")
  .replace(/&lt;/g, "<")
  .replace(/&gt;/g, ">")

return {
  ...item,
  _highlightResult,
  url: `${a.pathname}${hash}`,
}
```

**Comment by mochi-sann** [comment]
Thank you ,review
I've reduced the code as you pointed out.

**Pull request merged**